src/protractor.conf.js

```diff
@@ -60,6 +79,12 @@ if (config.headless) {
     '--headless',
     `--window-size=${config.browserWidth}x${config.browserHeight}`
   ];
+
+  firefoxConfig['moz:firefoxOptions'].args = [
```

Comment:
I don't like this idea. It's getting out of control. Imagine we will add safari and ie soon, so we will have to add another global config ? We definitely need to polish this into some high quality code. Maybe some factory?